libs/forms/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@studiohyperdrive/ngx-forms",
 	"homepage": "https://github.com/studiohyperdrive/ngx-tools/blob/master/projects/forms/README.md",
-	"version": "17.6.0",
+	"version": "17.6.1",
```

Comment:
Reminder: Make sure to wait with your version bump until after the PR has been reviewed.